docs/zk2kraft.html

```diff
@@ -148,7 +145,7 @@ <h5 class="anchor-heading">Removal Zookeeper configs</h5>
             </ul>
         </li>
     </ul>
-    <h5 class="anchor-heading">Removal metrics</h5>
+    <h2 class="anchor-heading">Removal Zookeeper Metrics</h2>
```

Comment:
Just a tiny suggestion to improve clarity:
```suggestion
    <h2 class="anchor-heading">Removing Zookeeper Metrics</h2>
```
or Zookeeper Metrics Removal or Removal of Zookeeper Metrics